src/platform/packages/shared/kbn-esql-ast/src/__tests__/context_fixtures.ts

```diff
@@ -214,5 +214,7 @@ export const getMockCallbacks = (): ICommandCallbacks => {
       .mockResolvedValue(expectedFields.map((name) => ({ label: name, text: name }))),
     getSuggestedUserDefinedColumnName: jest.fn(),
     getColumnsForQuery: jest.fn(),
+    hasMinimumLicenseRequired: jest.fn().mockReturnValue(true),
+    getActiveProduct: jest.fn(() => ({ type: 'observability', tier: 'complete' })),
```

Comment:
I wonder if this really needs to be a callback. Why don't we pass it as a context instead? It doesnt need to run as the hasMinimumLicenseRequired, it just returns a static value, no?

Reply:
yes, this was the first implementation I had done. But then I changed idea it because I wanted to maintain consistency with the work done before. I also wasn't sure if observability_tier was intended as our static context . 
I see there are more general contexts there